ansible_collections/arista/avd/roles/eos_validate_state/tasks/lldp_topology_no_fqdn.yml

```
@@ -23,6 +23,7 @@
     loop_var: ethernet_interface
   when: |
     (ethernet_interfaces is defined and ethernet_interfaces is not none) and
+   (ethernet_interface.shutdown is not arista.avd.defined(true)) and
```

```suggestion
    (ethernet_interface.value.shutdown is not arista.avd.defined(true)) and
```

Sorry, I provided wrong suggestion!

---

As well as the wrong indentation!